release_management/create_artifacts.py

```diff
@@ -90,15 +96,18 @@ def _stratisd_artifacts(namespace):
     if release_version != namespace.version:
         raise RuntimeError("Version mismatch.")
 
-    make_source_tarball("stratisd", release_version, output_abs_path)
+    r_v = ReleaseVersion(release_version, namespace.pre_release_suffix)
 
-    vendor_tarfile_name = vendor(manifest_abs_path, release_version)
+    if namespace.pre_release_suffix:
```

Comment:
Plz drop these two lines.

Reply:
I'm not sure of all of the lines you want me to delete, due to the way the diff is structured.

The intent of the `"Using suffix: " message here is to print the suffix used in the artifact build, in case it needs to be checked in an output log.  If this is still printed unconditionally, it would appear like this if executed without a suffix:
```
Using suffix: None
```

If the print() statement was removed entirely, the program would be less diagnosable.

What is your intent here?

Reply:
Removed.